app/config/routes.js

```diff
@@ -2,11 +2,11 @@ var React = require('react');
 var Main = require('../components/Main');
 var Home = require('../components/Home');
 var Router = require('react-router');
-var DefaultRoute = Router.DefaultRoute;
+var IndexRoute = Router.IndexRoute;
 var Route = Router.Route;
 
 module.exports = (
-  <Route name="app" path="/" handler={Main}>
-    <DefaultRoute handler={Home} />
+  <Route path="/" component={Main}>
+    <IndexRoute component={Home}/>
```

Comment:
I'm _pretty_ sure this is correct, basically identify `Home` as our default component inside of the `Main` wrapper.